src/v0/destinations/klaviyo/transform.js

```diff
@@ -56,7 +56,10 @@ const identifyRequestHandler = async (message, category, destination) => {
   }
   const traitsInfo = getFieldValueFromMessage(message, 'traits');
   const response = defaultRequestConfig();
-  const personId = await isProfileExist(message, destination);
+  let personId;
+  if (!mappedToDestination) {
```

Comment:
can we change this to contrext.channel == 'sources'. we have vdm and json mapper as well. this flag only works if retl connection is done by visual mapper

Reply:
Will this `channel === 'sources'` check only be sufficient or do we need both checks?

Reply:
just this will be sufficient

Reply:
Updated